### ISSUE TITLE
can't print because opc-life-over

### ISSUE THREAD

**Issue by biocyberman** [OpenPrinting/cups-filters#35]
I reported the problem here: https://github.com/apple/cups/issues/5295 but found out that is is better to report it on this repo. Essentially I want to continue printing despite the warning (warning about opc-life-over). How do I do that?

**Comment by biocyberman** [comment]
I think that is correct, because I tried installing cups-pdf and also failed when doing "Print Test Page" there, with the same error. The `opc-life-over` warning is just a red herring. 

As you pointed out, `pdftopdf` filter is complaining about missing `LIBQPDF_10` definition in `libqpdf.so.17`. I installed this `libqpdf-dev` and `libpqdf17` from the stock ubuntu package, which is equivalent to libqpdf version 6.0. So the problem is: 
Somehow `pdftopdf` is looking for `_ZN16QPDFObjectHandle9removeKeyERKNSt7__cxx1112basic_stringIcSt11char_traitsIcESaIcEEE` of `LIBQPDF_10` (libpqdf13). And libqpdf13 only exists until Ubuntu 14.04! Why does it require the old version of libqpdf13 when I already compiled with the newer version?

The similar pattern is found over here: https://stackoverflow.com/questions/36816570/glibcxx-3-4-21-not-defined-in-file-libstdc-so-6-with-link-time-reference

I updated the generated Makefile like this:
Old: 
`CXXFLAGS = -g -O2 -Wall  -D_GNU_SOURCE` 
New:
`CXXFLAGS = -g -O2 -Wall  -D_GNU_SOURCE -D_GLIBCXX_USE_CXX11_ABI=0 `
After this, the compilation process failed because it couldn't locate functions in libqpdf headers
